plugins/formal-verification/testData/diagnostics/no_contracts/extension_function.kt

```diff
@@ -3,3 +3,42 @@ fun Int.<!VIPER_TEXT!>inc<!>(): Int = this + 1
 fun <!VIPER_TEXT!>extension_method_call<!>() {
     3.inc()
 }
+
+class Foo
+fun Foo.<!VIPER_TEXT!>nothing<!>(): Unit { }
+fun Foo.<!VIPER_TEXT!>nothing2<!>(a: Int): Unit { }
+fun Foo.<!VIPER_TEXT!>inside<!>(): Unit {}
+
+class Bar() {
+    fun Foo.<!VIPER_TEXT!>inside<!>(): Unit { }
+    fun <!VIPER_TEXT!>driver<!>() {
+        val f = Foo()
+        f.inside()
+    }
+}
+
+class Baz() {
+    fun Foo.<!VIPER_TEXT!>inside<!>(): Unit { }
+    fun <!VIPER_TEXT!>driver<!>() {
+        val f = Foo()
+        f.inside()
+    }
+}
```

Comment:
How do `Baz` and `Bar` differ?

Reply:
They are the same! It was for testing name collision when generating the code for `Foo.inside()`.